self-core-impl/src/main/java/com/selfxdsd/core/BaseRepo.java

```diff
@@ -148,6 +148,18 @@ public Project activate() {
                 .replace("-", "")
         );
         wallets.activate(wallet);
+        final Contributors contributors = project.contributors();
+        contributors.register(
+            this.owner.username(),
+            this.owner.provider().name()
+        );
+        project.contracts().addContract(
+            project.repoFullName(),
+            this.owner.username(),
+            this.owner.provider().name(),
+            BigDecimal.valueOf(2500),
+            Contract.Roles.PO
+        );
```

Comment:
@criske I remember we once implemented a functionality, to automatically register the Contributor when adding a contract (if it's their first Contract in Self). Will not this work here? Maybe ``addContract(...)`` is enough. What do you think?

Reply:
@amihaiemil Yep, you're right, just checked the storage impl :D